openedx/core/djangoapps/geoinfo/api.py

```diff
@@ -22,7 +22,7 @@ def country_code_from_ip(ip_addr: str) -> str:
     try:
         response = reader.country(ip_addr)
         # pylint: disable=no-member
-        country_code = response.country.iso_code
+        country_code = response.country.iso_code or ""
```

Comment:
```suggestion
        country_code = getattr(response.country, "iso_code", "")
```
If we want to be extra safe.

Reply:
@navinkarkera, both `country` and `iso_code` are already in the class constructors within the `geoip2` code, so I think we don't need to be that safe here.